session_4/blog/app/views/articles/_card.html.erb

```diff
@@ -0,0 +1,21 @@
+
+  <div class="card" >
+  <img class="card-img" src="<%= url_for(article.image) %>" class="card-img-top" alt="...">
```

Comment:
Fill something in the alt-attribute

Reply:
Thank you for your suggestions. I have implemented some of them.
[https://github.com/hrushikeshj/IRIS-RoR-Bootcamp-2020/tree/session_4_hrushikesh/session_4#user-profile-page](https://github.com/hrushikeshj/IRIS-RoR-Bootcamp-2020/tree/session_4_hrushikesh/session_4#user-profile-page)